klyshko-operator/charts/klyshko/values.yaml

```diff
@@ -16,6 +16,7 @@ controller:
     repository: carbynestack/klyshko-operator-controller
     pullPolicy: IfNotPresent
     tag: latest
+  etcdEndpoint: 172.18.1.129:2379
```

Comment:
This variable has to be added to the table in the chart README.

Reply:
done